src/main/java/apoc/nodes/Nodes.java

```diff
@@ -303,6 +299,39 @@ public boolean isDense(@Name("node") Node node) {
         }
     }
 
+    @Procedure(value = "apoc.remove.nodes.withlabels", mode = Mode.WRITE)
```

Comment:
don't use remove as namespace, before nodes, but instead just call this `apoc.nodes.remove`